setup.cfg

```diff
@@ -63,6 +62,7 @@ install_requires =
     tqdm>=4.66.2,<5.0
     async-lru~=2.0.4
     psutil~=5.9.8
+    importlib_resources~=6.0.0
```

Comment:
Why is this needed over `import importlib.resources`?

Reply:
I did that because of this [comment](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/97085dea6ae23478e1021c27938992cf439a62f4/synapseclient/core/version_check.py#L168), but maybe that was a typo and it was meant to still be importlib.resources?

Reply:
Let's see if we can continue to use the existing library without adding extra dependencies. I was looking at the pypi page it was mentioned it was for backporting functionality for old python versions, which isn't the situation we are in.